recipe/meta.yaml

```diff
@@ -110,7 +111,7 @@ outputs:
       run:
         - {{ pin_subpackage('openmpi', exact=True) }}
         # host C compilers work fine (better) on mac
-        - {{ compiler('cxx') }}
+        - {{ compiler('cxx').replace(' ', '>=', 1) }}
```

Comment:
```suggestion
        - {{ compiler('cxx') | replace(' ', '>=', 1) }}
```

Reply:
I figured I'd try both syntaxes, since I have no clue and you seemed unsure.

Reply:
Indeed, I was just guessing... :sweat_smile: